doc-content/enterprise-only/employee-rostering/er-employees-proc.adoc

```diff
@@ -1,20 +1,21 @@
 [id='er-employees-proc']
 = Entering the list of employees
 
-You must enter the list of all employees of the business and the skills they possess. The application rosters these employees to spots according to their skills.
+You must enter the list of all employees of the business, the skills they possess, and the contracts that apply to them. The application rosters these employees to spots according to their skills, observing the work time limits in the contracts.
 
 .Procedure
 
 . To enter or change the list of employees, click the *Employees* tab.
 +
-You can see the numbers of currently visible employees in the top right part of the browser window, for example, `1-15 of 34`. You can use the *<* and *>* buttons to display other employees in the list. 
+You can see the numbers of currently visible employees in the top right part of the browser window, for example, `1-15 of 34`. You can use the *<* and *>* buttons to display other employees in the list.
 +
-You can enter any part of an employee name in the *Search* box to search for employees. 
+You can enter any part of an employee name in the *Search* box to search for employees.
 +
 . Complete the following steps to add a new employee:
 .. Click *Add Employee*.
 .. Enter the name of the employee into the text field under *Employee name*.
 .. Optionally, select one or more skills from the drop-down list under *Skill set*.
+.. Select a contract from the drop-down list under *Contract*.
```

Comment:
Consider whether you really need to specify drop-down. IBMSG:
drop-down adj
Use only if you must specify the type of menu or list.

Reply:
I do think it is important to get the user to click and see it's drop-down. Just "the list" could cause a "what list? I don't see a list here" reaction.

Reply:
OK